### PR TITLE
ready to merge: split bleeding diathesis due to thromboxane synthesis deficiency

### LINKED ISSUE #6419: split 'bleeding diathesis due to thromboxane synthesis deficiency'
MONDO:0013524
'bleeding diathesis due to thromboxane synthesis deficiency'

The orphanet x-ref (Orphanet:220443) refers to the disease
The OMIM x-ref (OMIM:614009) refers to the disease susceptibility. 

I think we need to "split" this term and have a 'disease' and a 'disease susceptibility term. 

 @nicolevasilevsky do you agree? 

Unfortunately, we will probably have to obsolete this term and create 2 new one because there is currently no clear meaning for MONDO:0013524

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #6419

### CODE REVIEW

**Comment by sabrinatoro** [comment]
@nicolevasilevsky I think https://github.com/monarch-initiative/mondo/issues/6546 was supposed to be added to https://github.com/monarch-initiative/mondo/pull/6895 and not this PR.

**Comment by nicolevasilevsky** [comment]
you are right! I will update this

**Comment by sabrinatoro** [comment]
Maybe you meant: https://github.com/monarch-initiative/mondo/issues/6897

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -269691,25 +269691,11 @@ property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/11008/nestor
...
-is_a: MONDO:0000009 {source="DC-OMIM:614009", source="MONDO:Redundant", source="OMIM:614009"} ! inherited bleeding disorder, platelet-type
-is_a: MONDO:0020573 {source="OMIM:614009"} ! inherited disease susceptibility
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/11608 {source="MONDO:mim2gene_medgen"} ! TBXA2R
-property_value: confidence "2.692307692307692" xsd:double
+name: obsolete bleeding diathesis due to thromboxane synthesis deficiency
+property_value: IAO:0000231 IAO:0000229
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6419#issuecomment-1615253065" xsd:anyURI
-property_value: IAO:0006012 "2023-11-01" xsd:string
+is_obsolete: true
+replaced_by: MONDO:0800446
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:269698]: This should be "consider" with both new IDs (because this obsoleted term represents 2 different terms)

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@nicolevasilevsky 
Only one change requested. Please see comment.

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:269698]: ah, right, thanks for catching that

**Pull request merged**